### DISCUSSION TITLE
Good First PR

### DISCUSSION THREAD

**Discussion by projectkepler-ru** [Bubberstation/Bubberstation#1965] [codebase discussion]
1. Put All Trouser into loadout and role restrict them accordingly. This should be done on skyrat
2. [Pride Pin?](https://github.com/Bubberstation/Bubberstation/issues/1723)
3. Rename hemophage organ on skyrat to avoid confusion
4. Add the new assistant hud that aeri made for TG
5. Blank Syndicate Bomb that can be purchased
6. add plumbing tool to cargo lathe

**Comment by sippykot** [comment]
assistant hud?
